data-visualize.qmd

```diff
@@ -651,36 +646,32 @@ ggplot(penguins, aes(x = species, y = body_mass_g)) +
   geom_boxplot()
 ```
 
-Alternatively, we can make frequency polygons with `geom_freqpoly()`.
-`geom_freqpoly()` performs the same calculation as `geom_histogram()`, but instead of displaying the counts with bars, it uses lines instead.
-It's much easier to understand overlapping lines than bars of `geom_histogram()`.
-There are a few challenges with this type of plot, which we will come back to in @sec-cat-num on exploring the covariation between a categorical and a numerical variable.
+Alternatively, we can make density plots with `geom_density()`.
```

Comment:
FWIW I really prefer frequency polygons to density plots.

If you do keep density plots, I think you need to say something like "a density plot is a type of smoothed histogram" and provide some justification as to why you'd use it.